### DISCUSSION TITLE
Welcome to fsociety Discussions!

### DISCUSSION THREAD

**Discussion by thehappydinoa** [fsociety-team/fsociety#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TobiTenno** [comment]
Would there be any desire for splitting installs based on package manager? I was looking to use nmap, but I use an arch distro, so obviously the "linux" command won't work for me. Are you open to having maps for separating apt-centric installs?

**Reply by thehappydinoa** [reply]
Hey there, that could be really cool especially for apt, brew, and maybe yum? I also don't think it sould be too difficult to implement.

**Reply by TobiTenno** [reply]
yeah, i think it's possible if you flex how the config is generated

**Reply by TobiTenno** [reply]
i've got a pr up #75

**Reply by TobiTenno** [reply]
is that an ok route to you?

**Reply by thehappydinoa** [reply]
Yup it has been merged, I also updated a couple other various things.

**Reply by TobiTenno** [reply]
cool!
